example/main/example_cli.cli

```diff
@@ -71,6 +71,14 @@ debug("Debugging parts of the system"){
 shell("System command"), cli_start_shell("bash");{
   <source:rest>("Single shell command"), cli_start_shell("bash");
 }
+
+run_program_err("Run program"), cli_start_program();
```

Comment:
The primary purpose of the example application is for the test-suite (under test/). 
The present CLI test command is referenced in test/test_cli.sh for example.
Other applications are in the separate repo https://github.com/clicon/clixon-examples
In my view this is primarily a "user-guide" example, ie how to run a python script from the CLI.
Therefore, if there is no regression test for this, the added commands do not belong here.
I think the best place for them are adding them to the CLI section of the user-guide instead, describing how a script could be started from the CLI.
The place to add such a subsection would be here: https://github.com/clicon/clixon-docs/blob/master/cli.rst
That said, if this command can replace the existing cli_start_shell command, one could replace it, and also change the test case in test_cli.sh so that it runs this new command instead.

To add such a section,

Reply:
remove in examle.cli